ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/IntegrationTestBase.java

```diff
@@ -195,6 +185,9 @@ public void setup() {
   public static void teardown() {
     clientAndServer.stop();
     runner.stop();
+    clientAndServer = null;
+    runner = null;
+    System.gc();
```

Comment:
You should never call `System.gc()`.  It's usually does nothing anyway - sometimes it triggers an expensive full GC but it will never reclaim memory that wouldn't be reclaimed if needed anyway.

Nulling out the static variables is likely to help since statics are GC roots so it won't be GC'd otherwise.